src/fs/read_dir.rs

```diff
@@ -57,25 +64,57 @@ pub async fn read_dir<'js>(
     path: String,
     options: Opt<Object<'js>>,
 ) -> Result<ReadDir> {
+    let mut path = path;
     let with_file_types = options
```

Comment:
When we now have two options. We can first check if we have options and then access the variables:
```rust
if let Some(opts) = opts.0{
 //check for withFileTypes & recursive option
}
```